### PR TITLE
Improve Kie Server Controller tests stability

### PR DESCRIPTION
**Pull request by jakubschwan**
testUpdateContainerNonValidReleaseId() sometimes failed on Eap 7 container due race condition. Test tried to get message from controller about update, but this message was not yet in controller. For avoid this random failing is added synchronization for Messages.

### CODE REVIEW

File: kie-server-parent/kie-server-tests/kie-server-integ-tests-common/src/main/java/org/kie/server/integrationtests/shared/KieServerSynchronization.java [hunk 1]
```diff
@@ -92,6 +93,27 @@ public static void waitForKieServerSynchronization(final KieServicesClient clien
...
+        waitForCondition(() -> {
+            ServiceResponse<KieContainerResource> containerResponse = client.getContainerInfo(containerId);
+            if (!containerResponse.getType().equals(ServiceResponse.ResponseType.SUCCESS)) {
+                return false;
+            }
+
+            // Kie Container store only one message
+            List<Message> messagesList = containerResponse.getResult().getMessages();
+            if (messagesList.size() == 1) {
+                for (String message : messagesList.get(0).getMessages()) {
```
Comment (sutaakar) [inline kie-server-parent/kie-server-tests/kie-server-integ-tests-common/src/main/java/org/kie/server/integrationtests/shared/KieServerSynchronization.java]: Just an idea, this cycle can be replaced by "return messagesList.get(0).getMessages().stream().anyMatch(n -> n.equals(expectedMessage));"

**Comment by mswiderski** [comment]
Are there any new changes expected or is this ready to be merged?

**Comment by mswiderski** [comment]
@jakubschwan any comment?

**Comment by jakubschwan** [comment]
I replaced cycle. Now is this PR ready to be merged

**Comment by mswiderski** [comment]
jenkins retest this

**Pull request merged**